### ISSUE TITLE
add raspbee lights to the supported template list

### ISSUE THREAD

**Issue by treban** [michbeck100/pimatic-hap#80]
https://forum.pimatic.org/topic/4243/the-big-zigbee-topic/33

add the device templates from the pimatic-raspbee plugin to the knownTemplates list

**Comment by 00lex** [comment]
I'd appreciate it if that happened. 
thank you

**Comment by michbeck100** [comment]
@treban did you check if the rasbpee plugin works if enabled?

**Comment by treban** [comment]
No.  I'm not using the Hap-Plugin.  Because I don't have iOS. But I will test it next week with a device.

**Comment by 00lex** [comment]
If you run MacOS Mojave, you can use HomeKit too

**Comment by 00lex** [comment]
any progress?

**Comment by michbeck100** [comment]
@00lex Not yet, but you could do me a favour and add these templates to the supported templates list:

```
      'raspbee-dimmer': DimmerAccessory
      'raspbee-ct': DimmerAccessory
      'raspbee-rgb': DimmerAccessory
```
Just add these lines at https://github.com/michbeck100/pimatic-hap/blob/master/hap.coffee#L60, restart pimatic and see if you can switch and dim your lights.

**Comment by michbeck100** [comment]
Please report back if the plugin works, then I will add them to the list "officially"

**Comment by 00lex** [comment]
Yes! Now I can see the bulbs in HomeKit but I can only switch them and change the brightness. 
not the color

**Comment by michbeck100** [comment]
Thats because they are just a `DimmerAccessory`. Since every plugin for coloured bulbs does the color calculation differently, I just added them as dimmers for now. I will check pimatic-raspbee can support more

**Comment by michbeck100** [comment]
BTW: All devices with type `RaspBeeSwitch` should work OOB as switches

**Comment by 00lex** [comment]
> Thats because they are just a `DimmerAccessory`. Since every plugin for coloured bulbs does the color calculation differently, I just added them as dimmers for now. I will check pimatic-raspbee can support more

I know. until them I will use hue-zll plugin for bulbs which doesn't get an update since 2 years. for the moment its okay but I think raspbee is the future for a zigbee environment and pimatic.   
if I can do something, say it. thank you for feedback

**Comment by treban** [comment]
i will implement two new functions on the raspbee side so the 
HueLightAccessory could be used for RGB

**Comment by scobby** [comment]
I have many different zigbee devices. Maybe I can do some tests if the changes are implemented.

**Comment by michbeck100** [comment]
If you know how to install the latest master you can go ahead and test. Changes were implemented with https://github.com/michbeck100/pimatic-hap/commit/b4e6efa1f60aef65382eb8ef37596950c671af3a. RGB lights will be next once @treban is ready.

**Comment by treban** [comment]
@scobby 

the new raspbee version **0.1.0-beta.1**  supports now HAP Integration with RGB capabilities 
(and some more nice features) [readme](https://github.com/treban/pimatic-raspbee/blob/master/README.md)
maybe you can test it?

please install the beta from plugin-rasbpee and change the accessory for raspbee-rgb
[manual install](https://pimatic.teamemo.com/Usage/Installing-Plugins#rtBJf1n1Lial)
`sudo npm install --unsafe-perm pimatic-pluginname`
inside your pimatic-app directory.

https://github.com/michbeck100/pimatic-hap/blob/master/hap.coffee#L60
'raspbee-dimmer': DimmerAccessory
'raspbee-ct': DimmerAccessory
'raspbee-rgb': **HueLightAccessory**

**Comment by michbeck100** [comment]
@scobby the raspbee plugin is released in version 0.1.0 now. Did you test if the `HueLightAccessory` works for you?

**Comment by 00lex** [comment]
The result is the same as with `'raspbee-rgb': DimmerAccessory`
- white hue bulbs are 100% compatible
- ambient white ist available, but without support for color temperature  
- color bulbs doesn't show up in HomeKit

**Comment by michbeck100** [comment]
@00lex Make sure to restart pimatic. Maybe you even have to import your devices in the homekit app again. You could test this by creating a testing home first.

**Comment by 00lex** [comment]
got it! the discovery doesn't work correct. the auto recognition for color bulbs and light stripes recognizes RGBCT and it doesnt work show up in HomeKit. if I change Raspbee-RGBCT to Raspbee-RGB it works both kind of lights show up with full support :)

now we just need a fix for ambient-white bulbs

**Comment by michbeck100** [comment]
I guess that's because the template name of Raspbee-RGBCT is `raspbee-rgbct` and not 'raspbee-rgb'. see https://github.com/treban/pimatic-raspbee/blob/master/raspbee.coffee#L1187

The Raspbee-RGBCT device might work too, just try to add another row with `'raspbee-rgbct': HueLightAccessory` at https://github.com/michbeck100/pimatic-hap/blob/master/hap.coffee#L60

**Comment by 00lex** [comment]
gesagt, getan. the solution was obvious. 
now with additional `'raspbee-rgbct': HueLightAccessory` color things work. 

the only open point is color temperature of ambient white bulbs. its not a big thing for me

**Comment by 00lex** [comment]
can you give me a hint how to set a color with rgbct? 
with hue-zll I wrote `set color of Paradieslicht to hue 12.85 sat 55`

i don't understand why. the variables are nearly same:
<img width="1670" alt="Bildschirmfoto 2019-07-15 um 18 14 02" src="https://user-images.githubusercontent.com/39100562/61231530-97482c80-a72c-11e9-8bb6-20de1369b26d.png">

**Comment by michbeck100** [comment]
Not sure where you wrote that. I you are talking about the pimatic rules section, than you have to ask the developer of pimatic-raspbee (which is @treban I think)

**Comment by 00lex** [comment]
my rule is with hue-zll:
`if PIR-x is present then set color of Paradieslicht to hue 12.85 sat 55`
and I don't know how to translate this to raspbee. I will ask him. i've already bothered him enough :)

**Comment by michbeck100** [comment]
Closing this as support was released with 0.12.0
